assets/RNAseq_report.Rmd

```diff
@@ -130,16 +130,17 @@ if(file.exists(QAfile)){
     df_QA <- read.table(QAfile, header=TRUE, sep="\t", check.names = FALSE)
     conditions <- grepl(colnames(df_QA),pattern = "Condition")
     df_QA_cond <- as.data.frame(df_QA[ ,conditions])
-    colnames(df_QA_cond) <- colnames(df_QA)[conditions]
-    df_QA_neat <- df_QA[,c("QBiC Code", "Secondary Name", "Lab ID", "Sample Type")]
+    colnames(df_QA_cond) <- colnames(df_QA)[conditions] 
```

Comment:
In the lines 133 und 137 trailing whitespaces need to be removed to make the linting pass.